tests/blas_like/ColumnNorms.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+   Copyright (c) 2009-2016, Jack Poulson
+   All rights reserved.
+
+   This file is part of Elemental and is under the BSD 2-Clause License,
+   which can be found in the LICENSE file in the root directory, or at
+   http://opensource.org/licenses/BSD-2-Clause
+*/
+
+#include <El.hpp>
+using namespace El;
+
+template <typename T, DistWrap W>
+void TestColumnTwoNorms(Int m, Int n, const Grid& g, bool print) {
+  // Generate random matrix to test.
+  DistMatrix<T, MC, MR, W> A(g);
+  Uniform(A, m, n);
+  if (print) {
+    Print(A, "A");
+  }
+  DistMatrix<T, MR, STAR, W> norms(g);
+  ColumnTwoNorms(A, norms);
+  if (print) {
+    Print(norms, "norms");
+  }
+  for (Int j = 0; j < A.LocalWidth(); ++j) {
+    T got = norms.GetLocal(j, 0);
+    T expected = 0;
+    for (Int i = 0; i < A.LocalHeight(); ++i) {
+      T val = A.GetLocal(i, j);
+      expected += val * val;
+    }
+    expected = mpi::AllReduce(expected, g.ColComm());
+    expected = Sqrt(expected);
+    if (Abs(got - expected) > 1e-5) {
```

Comment:
It would be more rigorous to use a bound of the form `n * limits::Epsilon<El::Base<T>>()` times a small constant (e.g., 10).